en/dev_setup/dev_env_linux_ubuntu.md

```diff
@@ -176,13 +176,14 @@ Setup instructions for *Snapdragon Flight* are provided in the flight controller
 * [Configuration](../flight_controller/snapdragon_flight_configuration.md)
 
 
+<a id="fast_dds"></a>
 <a id="fast_rtps"></a>
```

Comment:
FYI only, I restored this. Doesn't hurt and means that external links to page will not be broken.

Reply:
That needs to be reverted above as well then

Reply:
No, it is OK above. The only reason to keep both links is so that anything linking here from outside our docs still goes to the right place.